deadbelly/breathEZ
# Feature/render routes

### What does this PR do? 
>Clearly and concisely describe the feature. 
- adds router import to `App`
- adds routes to `App`
- removes need for state to track which view is current
  - this is done by taking advantage of the url's directly
- cleans up some warnings from the console
- leaves a comment for a warning in `Faq.js`
### Is this a feature or Fix/Refactor?  
- feature
### Problems Encountered & Resulting Solutions  
>State any problems encountered 
- There were no major issues.
>Describe your code changes in detail for reviewers.
- imported route
- implemented two strict routes, one for the main view and one for the faq
- removed responsibility to track views from state of `App`
- passed path from inherent `props` from `App` to `Header`
  - this allows for the app to render the proper path due to the url itself
- created `Links` with classNames to make styling easier in the future
>A couple warnings that I cleaned up
- removed unused variables from `Selector`
- added unique keys to cities and states
### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
- `src/Components/App/App.js`
- `src/Components/Header/Header.js`
### How should this be tested?
>What steps need to be taken to confirm functionality? 
- pull down branch (to test branch, not main)
- run `npm start`
- click on `faq` link
   - observe moving to the faq
 - click on `return home` link
   - observe returning home
### Steps To Take
>Any questions and possible steps moving forward.
- One warning that is still there is from `faq.js`:
  -  a `<ul>` cannot be the child of a `<p>`
### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
- This closes #30
